client/src/services/unoService.js

```diff
@@ -17,6 +17,15 @@ export default {
     return BaseService.get(`/api/games/${gameId}`);
   },
 
+  async getPlayerNameFromToken() {
```

Comment:
Interesting.. is this getting you own personal player name?

Reply:
If so. I think we can modify the gameState to bring back `player_name` in that object. It would be a pretty simple enhancement and wouldn't require a second call back on the database.